features/step_definitions/add_warning_to_tools_steps.rb

```diff
@@ -44,3 +38,12 @@
     expect(page).to_not have_content('Mae’r offeryn hwn yn cael ei ddiweddaru ar hyn o bryd ar gyfer y flwyddyn dreth newydd.')
   end
 end
+
+private
+def retirement_option_pages(locale, tool)
+  if locale == 'en'
+    rio_page.load(locale: 'en', tool_name: 'retirement-income-options', id: tool)
+  else
+    rio_page.load(locale: locale, tool_name: 'opsiynau-incwm-ymddeoliad', id: tool)
+  end
+end
```

Comment:
This 'cases' and 'if' on cucumber specs sounds like a code smell. Question, those warnings will be temporarily?